crt_portal/cts_forms/forms.py

```diff
@@ -2553,6 +2559,12 @@ def update(self, reports, user):
                 'description': d
             } for (v, d) in self.get_actions(report)])
 
+        if tags:
+            for report in reports:
+                for tag in tags:
+                    report.tags.add(tag)
+                    activities.append({'user': user, 'report': report, 'verb': 'Added tag: ', 'description': tag.name})
+
```

Comment:
(since the for loop won't run if tags is empty, you can flip the tags / reports and lose the `if`)

```suggestion
      for tag in tags:
          for report in reports:
              report.tags.add(tag)
              activities.append({'user': user, 'report': report, 'verb': 'Added tag: ', 'description': tag.name})
```

Reply:
Tried this but without the if statement I got an error that tags wasn't iterable